### DISCUSSION TITLE
anyway I can set a blank new tab page?

### DISCUSSION THREAD

**Discussion by SamilKaraduman** [Eloston/ungoogled-chromium#1967] [Q&A] [answered]
i dont want my history to be shown on a new tab page. and no, a 3rd party extension is not an option.
<img width="1429" alt="Screen Shot 2022-05-21 at 12 00 44 PM" src="https://user-images.githubusercontent.com/83664323/169659790-ff268f44-1085-49ad-b170-ac2b8326b309.png">

**Comment by networkException** [comment, marked as answer]
See `--custom-ntp` documented here: https://github.com/Eloston/ungoogled-chromium/blob/master/docs/flags.md#available-only-on-desktop=

**Reply by SamilKaraduman** [reply]
i dont see anything. is there somethng im doing wrong?
<img width="1431" alt="Screen Shot 2022-05-21 at 8 18 20 PM" src="https://user-images.githubusercontent.com/83664323/169673110-c7d547f0-908e-4575-86d0-003f9f466cd1.png">

**Reply by tomasz1986** [reply]
Yes, please try searching without the `--`.

**Reply by SamilKaraduman** [reply]
yeah... still doesnt work.

**Reply by tomasz1986** [reply]
Searching for just `ntp` should also do it…

For the record, this is what you're supposed to see.

![image](https://user-images.githubusercontent.com/5626656/169673647-982d8811-ec6c-4408-bb2a-70a9b10ab277.png)

**Reply by SamilKaraduman** [reply]
<img width="1415" alt="Screen Shot 2022-05-21 at 9 37 35 PM" src="https://user-images.githubusercontent.com/83664323/169674681-87641483-941b-43cc-b932-4d1c199dae35.png">
tried that already... thank you for trying to help though.

**Reply by jstkdng** [reply]
It should show up, here's the flag under arch linux.
![image](https://user-images.githubusercontent.com/52050345/169675163-8f06b595-b8ce-4cf0-b040-20f78934f1f5.png)
What version are you using and where did you get it?

**Reply by SamilKaraduman** [reply]
got it from here: https://ungoogled-software.github.io/ungoogled-chromium-binaries/

im using an apple silicon based mac, and the version i was using was 92.0.4515.131-1

**Reply by jstkdng** [reply]
wow, that's a really old version, the patch mentioned was added around version 96.x. Try updating with this: https://github.com/kramred/ungoogled-chromium-macos/releases

There are releases for arm and x86 macs

**Reply by SamilKaraduman** [reply]
thank you! this really helped.

**Comment by jacob755** [comment]
you need this flag --disable-top-sites
or in chrome://flags/  search for Custom New Tab Page
and set like this file:///C:/Chromium/Extension/Tabs/blank.htm
